runtime/android/core_internal_empty/BUILD.gn

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+
+# The "xwalk_core_internal_empty_embedder_apk" is used for generating R.class
+# for library_part.jar in resource_map
+android_apk("xwalk_core_internal_empty_embedder_apk") {
+  apk_name = "XWalkCoreInternalEmptyEmbedder"
+  android_manifest = "AndroidManifest.xml"
+  native_libs = [ "$root_out_dir/libxwalkdummy.so" ]
+  deps = [
+    "//base:base_java",
+    "//components/web_contents_delegate_android:web_contents_delegate_android_java_resources",
+    "//content/public/android:content_java_resources",
+    "//ui/android:ui_java_resources",
+    "//xwalk/runtime/android/core_internal:xwalk_core_internal_java_resources",
+    "//xwalk/runtime/android/dummy_lib:libxwalkdummy",
+  ]
+}
+
+# "xwalk_core_empty_embedder_apk" is used for generating the R.txt.
+android_apk("xwalk_core_empty_embedder_apk") {
+  apk_name = "XWalkCoreEmptyEmbedder"
+  android_manifest = "AndroidManifest.xml"
+  deps = [
+    "//components/web_contents_delegate_android:web_contents_delegate_android_java_resources",
+    "//content/public/android:content_java_resources",
+    "//ui/android:ui_java_resources",
+    "//xwalk/runtime/android/core:xwalk_core_java_resources",
+    "//xwalk/runtime/android/core_internal:xwalk_core_internal_java_resources",
```

Comment:
I don't think it's needed (which also invalidates the comment above the target), see #3880.